src/processing/CommunicateChange.java

```diff
@@ -0,0 +1,90 @@
+package processing;
+
+import java.util.*;
+import processing.IChanges;
+import processing.utility.*;
+
+public class CommunicateChange {
+	
+	// identifier to IChanges handler map
+    // private Map <Identifier, IChanges> identifierTohandler;
+    
+	/*
+     * Computes the set of pixels which will have effect
+     * on the screen and notify the UI using the getChanges
+     * method of IChanges.
+     */
+	
+	// oldPixels -> previous pixels of the object
+	// newPixels -> new pixels of the object
+	public static void provideChanges(
+			Identifier identifier, 
+			ArrayList<Pixel> oldPixels, 
+			ArrayList<Pixel> newPixels) {
+		
```

Comment:
multiple input parameters in separate lines? Check once with the Google code conventions.

Reply:
There is no convention regarding this. So, it's fine.